omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -193,156 +197,115 @@
                         </td>
                     </tr>
                     <tr>
                         <td>
                             <uimessage code="mirebalais.vitals.o2sat.title"/>
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').obsDatetime"/>
-                            </p>
-                        </td>
-                        <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').valueNumeric"/>
-                            </p>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <uimessage code="pihcore.lab.isFastingGlucose"/>
-                        </td>
-                        <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:Fasting for blood glucose test').obsDatetime"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').obsDatetime"/>
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:Fasting for blood glucose test').valueCoded.name"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').valueNumeric.intValue()"/>
+                            <span class="light-font">
+                                <lookup complexExpression="#if($fn.latestObs('PIH:BLOOD OXYGEN SATURATION')) % #end" />
+                            </span>
                         </td>
                     </tr>
                     <tr>
                         <td>
                             <uimessage code="pihcore.lab.glucose"/>
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:SERUM GLUCOSE').obsDatetime"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:SERUM GLUCOSE').obsDatetime"/>
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:SERUM GLUCOSE').valueNumeric"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:SERUM GLUCOSE').valueNumeric.intValue()"/>
+                            <span class="light-font">
+                                <lookup complexExpression="#if($fn.latestObs('PIH:Fasting for blood glucose test')) #if($fn.latestObs('PIH:Fasting for blood glucose test').valueCoded.name != 'No') (en ayuno) #else (sin ayuno) #end #end" />
```

Comment:
Unfortunately hard-codes some things that should be message properties ("en ayuno", "sin ayuno"), but oh well. Velocity.